src/query/hiro-config/hiro-config.query.ts

```diff
@@ -0,0 +1,46 @@
+import { GITHUB_ORG, GITHUB_PRIMARY_BRANCH, GITHUB_REPO } from '@common/constants';
+import { useQuery } from 'react-query';
+
+export interface HiroMessage {
+  title: string;
+  text: string;
+  purpose: 'error' | 'info' | 'warning';
+  publishedAt: string;
+  dismissible: boolean;
+  learnMoreUrl?: string;
+}
+
+interface HiroConfig {
+  messages: any;
+  activeFiatProviders: any;
```

Comment:
We could type this better, we know that it's keyd by string, and the type of the key's value:

```suggestion
  activeFiatProviders: Record<string, TheActiveFiatProviderType>;
```